DQMOffline/RecoB/python/PixelVertexMonitor_cff.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
I guess this file entered here because it used to be modified https://github.com/cms-patatrack/cmssw/pull/548 and I had asked earlier Andrea to integrate the commits from that PR here for being able to run the validation in a self-contained way. It's not strictly necessary to validate Pixel local reconstruction, so I think indeed can be dropped here and maintained in #31723

Reply:
if we are to remove this file is there a single runTheMatrix workflow using GPU that still works?

Reply:
@slava77 as @mmusich said above, the main objective of [patatrack#548](https://github.com/cms-patatrack/cmssw/pull/548/) was to add the Picel local reco validation sequence. So, even if this file is removed, the local reco validation sequence will run for any Pixel only GPU wf, e.g 11634.502. The relevant config files needed are:-
Validation/Configuration/python/globalValidation_cff.py
Validation/Configuration/python/postValidation_cff.py
Validation/SiPixelPhase1ConfigV/python/SiPixelPhase1OfflineDQM_sourceV_cff.py

Reply:
OK, I guess I can move it back to #31723.
Will do when I regenerate the PR to include the fixes from the Patatrack branch.